subprojects/gradle/cd/src/test/kotlin/com/avito/ci/steps/PerformanceIntegrationTest.kt

```diff
@@ -32,7 +33,7 @@ class PerformanceIntegrationTest {
                         "com.avito.android.cd"
                     ),
                     customScript = """
-                         ${registerUiTestConfigurations(performanceConfigurationName)}
+                         ${registerUiTestConfigurations(performanceConfigurationName, isPerformance = true)}
```

Comment:
Let's remove everything related to performance plugin

Reply:
will do in next release; it's also a bad practice to do such thing in unrelatable PR